chains/orchestrator-relays/runtime/dancelight/src/bridge_to_ethereum_config.rs

```diff
@@ -98,88 +98,8 @@ impl snowbridge_pallet_outbound_queue::Config for Runtime {
     type OnNewCommitment = CommitmentRecorder;
 }
 
-// Very stupid, but benchmarks are written assuming a fork eopch,
-// and test vectors assuming another one
-// We need allow dead code here because for regular builds this variable is not used
-// This variable is only used in test, fast-runtime or runtime-benchmarks
```

Comment:
Don't remove the comments, they give useful context

Reply:
sure, moved it to the separate file as well.